src/main/java/org/openstreetmap/atlas/utilities/command/subcommands/SubAtlasCommand.java

```diff
@@ -137,26 +120,26 @@ protected void processAtlas(final Atlas atlas, final String atlasFileName,
 
         if (this.optionAndArgumentDelegate.hasOption(SLICE_FIRST_OPTION_LONG))
         {
-            if (this.polygon.isPresent())
+            if (this.polygon != null)
```

Comment:
That seems odd, is it expected?

Reply:
It is. Before, I was storing this instance variable as an `Optional` which is bad style. I changed it to a plain polygon, so now it needs a null check.